index.ts

```diff
@@ -1071,9 +1072,10 @@ commander
         let coloredSeverity = ({
           ERROR: chalk.red(severity),
           INFO: chalk.blue(severity),
-          DEBUG: chalk.yellow(severity)
+          DEBUG: chalk.yellow(severity),
+          NOTICE: chalk.magenta(severity)
```

Comment:
(We should add [`trailing-comma`](https://palantir.github.io/tslint/rules/trailing-comma/) to our `tslint` to prevent 2 line diffs.)

Reply:
Noted, will put that in the next PR.